### DISCUSSION TITLE
Akka Persistence + IActoRef

### DISCUSSION THREAD

**Discussion by mrxrsd** [akkadotnet/akka.net#5139] [Q&A]
Hi,

I've successfully implemented akka persistence backed by redis, but now I'm trying to switch to another serializer instead of json. I'm using as a reference the msgPack example that is in the Akka.Persistence.Redis repository but I got an exception:
"FormatterNotRegisteredException: IActorRef is not registered in resolver: MessagePack.Resolvers.StandardResolver".

I've found this issue (https://github.com/akkadotnet/akka.net/issues/3811) and now I was in doubt if it is impossible to use or if I implement this "resolver" somehow will work.

Is it an option use Hyperion instead msgpack here?

tks

**Comment by Aaronontheweb** [comment]
You can use Hyperion with Redis - is that the thrust of your question?
